src/pretix/base/management/commands/export.py

```diff
@@ -60,7 +60,7 @@ def handle(self, *args, **options):
             sys.exit(1)
 
         locale = options.get("locale", None)
-        timezone = pytz.timezone(options['timezone']) if options.get('timezone') else None
+        timezone = pytz_deprecation_shim.timezone(options['timezone']) if options.get('timezone') else None
```

Comment:
Is this deprectation_shim here because there might be plugins with exports that expect pytz-timezones?